k8s/ithc/common/ia/home-office-integration-api.yaml

```diff
@@ -0,0 +1,31 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: ia-home-office-integration-api
+  namespace: ia
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: ia-home-office-integration-api
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: ia-home-office-integration-api
+    version: 0.0.12
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/ia/home-office-integration-api:latest
+      ingressHost: ia-home-office-integration-api-ithc.service.core-compute-ithc.internal
+      environment:
+        HOME_OFFICE_ENDPOINT: "https://cts-challenge.prp1.np.immigration-api.homeoffice.gov.uk"
```

Comment:
let's use mock api as for the first test, we know that Gareth is working to white list the cluster but it is not done yet

Reply:
changing to mock api in ithc